lib/src/package_config.dart

```diff
@@ -274,3 +276,24 @@ String extractLanguageVersion(VersionConstraint c) {
   }
   return '${minVersion.major}.${minVersion.minor}';
 }
+
+/// Parses the (now obsolete) .packages file.
+Map<String, Uri> parsePackagesFile(List<int> bytes, Uri base) {
+  final result = <String, Uri>{};
+  for (var line in utf8.decode(bytes).split('\n')) {
+    line = line.trim();
```

Comment:
I think this is too much... From what I can see the original code only ignores CR and LF:
https://github.com/dart-lang/package_config/blob/master/lib/packages_file.dart#L33-L97

Treating both of them as line breaks.. and it allows empty lines.